digos-ambassador/Content/Transformations/Species/snake/Leg.yml

```diff
@@ -0,0 +1,9 @@
+part: Leg
+species: snake
+description: "Transforms the target's {@side} leg into that of an anthro snake."
+default_base_colour: "dull grey"
+is_nsfw: false
+shift_message: "Smooth scales smother over {@target}'s {@side} leg. {@f|Their} muscles ripple and coil under {@f|their} skin, filling with potent and explosive muscle. Bones audibly crunch and pop as the arch of their foot stretches and their leg shifts into a predatory digitigrade stance, toes merging and melting into three large claw-tipped digits."
+grow_message: "The {@side} side of {@target}'s pelvis ripples and bulges, erupting into a digitigrade three-toed leg covered in smooth {@colour} scaled skin that sports rippling coils of muscle."
+single_description: "{@f|They have} a potent three-clawed digitigrade {@side} leg covered in smooth {@colour} scales, its densely coiled muscles visibly twitching with explosive leaping energy."
+uniform_description: "{@f|They have} potent three-clawed digitigrade legs covered in smooth {@colour} scales. Their densely coiled muscles visibly twitch with explosive leaping energy."
```

Comment:
No naga form in the uniform? We can always add a naga species and mix and match parts, of course :P

Reply:
Not sure. Felt like the right call at the time but it's an easy fix. Entirely up to you.